apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/Numeric.cs

```diff
@@ -575,55 +498,29 @@ public decimal ToDecimal(LossOfPrecisionHandling lossOfPrecisionHandling)
 
         /// <summary>
         /// Returns the result of adding two <see cref="NumericValue"/> values together.
+        /// Note that an assumption is made that both values either have the same <see cref="NumericType"/>
+        /// or at least one is <code>default(NumericValue)</code>.
         /// </summary>
         /// <param name="lhs">The first value to add.</param>
         /// <param name="rhs">The second value to add.</param>
         /// <returns>
-        /// The result of adding the two values. The range of the <see cref="NumericType"/> of the return value
-        /// will be the union of the ranges of the <see cref="NumericType"/>s from the two operands.
+        /// The result of adding the two values.
         /// </returns>
         public static NumericValue operator +(NumericValue lhs, NumericValue rhs)
-        {
-            if (!(NumericType.GetUnionType(lhs.Type, rhs.Type) is NumericType newType))
-            {
-                return default;
-            }
-            if (lhs.Type?.Scale == rhs.Type.Scale)
-            {
-                return new NumericValue(lhs._scaledValue + rhs._scaledValue, newType);
-            }
-            if (newType.Scale == lhs.Type?.Scale)
-            {
-                return new NumericValue(lhs._scaledValue + rhs.GetScaled(newType.Scale), newType);
-            }
-            return new NumericValue(lhs.GetScaled(newType.Scale) + rhs._scaledValue, newType);
-        }
+            => new NumericValue(lhs._scaledValue + rhs._scaledValue, lhs.Type ?? rhs.Type);
```

Comment:
Meganit: move the => to the end of the declaration line (and below)

Reply:
Done